lib/node_modules/@stdlib/stats/base/dists/weibull/skewness/benchmark/benchmark.native.js

```diff
@@ -0,0 +1,71 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2025 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+// MODULES //
+
+var resolve = require( 'path' ).resolve;
+var bench = require( '@stdlib/bench' );
+var Float64Array = require( '@stdlib/array/float64' );
+var randu = require( '@stdlib/random/base/randu' );
+var isnan = require( '@stdlib/math/base/assert/is-nan' );
+var tryRequire = require( '@stdlib/utils/try-require' );
+var pkg = require( './../package.json' ).name;
+
+
+// VARIABLES //
+
+var skewness = tryRequire( resolve( __dirname, './../lib/native.js' ) );
+var opts = {
+	'skip': ( skewness instanceof Error )
+};
+
+
+// MAIN //
+
+bench( pkg, opts, function benchmark( b ) {
+
+	var len;
+	var k;
+	var lambda;
+	var y;
+	var i;
+
+	len = 100;
+	k = new Float64Array( len );
+	lambda = new Float64Array( len );
+	for ( i = 0; i < len; i++ ) {
+		k[ i ] = ( randu() * 10.0 ) + 1.0; // shape parameter > 0
+		lambda[ i ] = ( randu() * 10.0 ) + 1.0; // scale parameter > 0
```

Comment:
You have to use `uniform` function to generate the parameters for added clarity as you did in `benchmark,c`
```suggestion
		k[ i ] = uniform( 1.0, 10.0 ); // shape parameter > 0
		lambda[ i ] = uniform( 1.0, 10.0 ); // scale parameter > 0
```